src/xkb/compose.rs

```diff
@@ -0,0 +1,95 @@
+use xkb::ffi::*;
+use std::ffi::CString;
+use super::{Context, Keysym};
+
+pub use xkb::ffi::xkb_compose_compile_flags as CompileFlags;
+pub use xkb::ffi::xkb_compose_state_flags as StateFlags;
+
+pub use xkb::ffi::xkb_compose_status as Status;
+pub use xkb::ffi::xkb_compose_feed_result as FeedResult;
+
+
+pub struct Table {
+  ptr: *mut xkb_compose_table,
+}
+
+impl Table {
+  pub fn new(context: &Context, locale: &str, flags: CompileFlags) -> Table {
+    let string = CString::new(locale).unwrap();
+
+    unsafe {
+      Table {
+        ptr: xkb_compose_table_new_from_locale(context.get_raw_ptr(), string.as_ptr(), flags)
+      }
+    }
+  }
+
```

Comment:
`new_from_locale` is actually `new` because that's what you want most of the time, I can add `from_buffer` tho.

`from_file` is probably useless since it would be easier to just use `from_buffer` than doing some weird wrapping with a reader.